cmd/restic/cmd_backup.go

```diff
@@ -115,6 +117,9 @@ func init() {
 	f.StringVar(&backupOptions.TimeStamp, "time", "", "time of the backup (ex. '2012-11-01 22:08:41') (default: now)")
 	f.BoolVar(&backupOptions.WithAtime, "with-atime", false, "store the atime for all files and directories")
 	f.BoolVar(&backupOptions.IgnoreInode, "ignore-inode", false, "ignore inode number changes when checking for modified files")
+	if runtime.GOOS == "windows" {
+		f.BoolVar(&backupOptions.UseWindowsVSS, "use-windows-vss", false, "use windows volume shadow copy service")
```

Comment:
Perhaps name this "windows-vss" instead of "use-windows-vss", for the purposes of being short and concise. Or "with-windows-vss".

Reply:
I kept that one open because this might need a bit of discussion.

Reply:
👍

Reply:
After mutiple reviews no one else complained. Can we close this keeping the current name?